content/guides/integrations/google_cloud/google_cloud_spanner/Java.md

```diff
@@ -205,14 +223,14 @@ public class SpannerOpenCensusTutorial {
     <maven.compiler.target>1.8</maven.compiler.target>
     <maven.compiler.source>1.8</maven.compiler.source>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <opencensus.version>0.11.0</opencensus.version>
+    <opencensus.version>0.18.0</opencensus.version>
   </properties>
 
   <dependencies>
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-spanner</artifactId>
-      <version>0.33.0-beta</version>
+      <version>1.5.0</version>
```

Comment:
Consider adding a version variable here as well.

Reply:
Roger that, thanks.